target/linux/mediatek/image/filogic.mk

```diff
@@ -472,6 +472,23 @@ define Device/cudy_re3000-v1
 endef
 TARGET_DEVICES += cudy_re3000-v1
 
+define Device/cudy_tr3000-v1
+  DEVICE_VENDOR := Cudy
+  DEVICE_MODEL := TR3000
+  DEVICE_VARIANT := v1
+  DEVICE_DTS := mt7981b-cudy-tr3000-v1
+  DEVICE_DTS_DIR := ../dts
+  SUPPORTED_DEVICES += R47
```

Comment:
Is this required?

Reply:
I am not really sure. I got the R47 code from a DTS that Cudy has published in https://www.cudy.com/openwrt_software_download

Other Cudy routers in OpenWrt have the same variable too (and I used their DTS as an example for this)

Reply:
I see, it's for compatibility on the firmware validation when upgrading from an OpenWrt image provided by Cudy.
That image for TR3000 v1 is not provided now, but if there is a possibility that it will be released in the future, I'm fine to keep this `SUPPORTED_DEVICES`.